vars/Amazon.yml

```diff
@@ -0,0 +1,23 @@
+---
+# A dictionary whose keys are values of the Ansible fact
+# ansible_architecture and whose values are processor architecture for
+# which Nessus is to be installed, as specified in the Nessus package
+# filename
+architectures:
+  arm64: aarch64
+  i386: i386
+  x86_64: x86_64
+
+# The format/file extension of the Nessus package file (e.g. deb, rpm)
+format: rpm
+
+# A regex that will match the name of the Nessus package file that
+# should be installed.  Note that we force the distribution name here,
+# because what Nessus uses ("amzn") does not match what Ansible uses
+# ("Amazon").
+package_file_regex: "Nessus-{{ version }}-amzn[0-9]*\\.{{ architectures[ansible_architecture] }}\\.{{ format }}"
```

Comment:
In all of the `package_file_regex` definitions you use the `*` operator in the distribution specifier. Are you intentionally using "zero or more" as opposed to using `+` for "one or more"?

Reply:
Yes.  My thinking was that they may at some point have a single `Nessus-*-debian-*.deb` file for a particular version that supports all extant releases of that distribution.  Are you thinking that this could result in an error?

Reply:
No I just wanted to make sure it was intentional since all of the file patterns are `distribution\d+` for Linux installers right now.